frontend/src/pages/instance/Settings/Editor.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <form class="space-y-6">
-        <TemplateSettingsEditor v-model="editable" :editTemplate="false" />
+        <TemplateSettingsEditor v-model="editable" :editTemplate="false" :team="team" />
```

Comment:
Ben, is there a commit missing from this PR? 

`TemplateSettingsEditor` does not have a `team` prop.

Reply:
never mind, i am blind